### ISSUE TITLE
TypeError: Cannot read property 'score' of undefined

### ISSUE THREAD

**Issue by BasvanH** [greenbone/gsa#3022]
### Expected behavior

Able to view an CVE report.

### Actual behavior

When viewing an CVE report you get this error in the browser. It's with all CVE based reports.
```
TypeError: Cannot read property 'score' of undefined
TypeError: Cannot read property 'score' of undefined
    at Function.parseElement (https://security.centralpoint.cloud/static/js/main.830b8bf3.chunk.js:1:57288)
    at Function.fromElement (https://security.centralpoint.cloud/static/js/main.830b8bf3.chunk.js:1:30710)
    at Function.parseElement (https://security.centralpoint.cloud/static/js/main.830b8bf3.chunk.js:1:83104)
    at Function.fromElement (https://security.centralpoint.cloud/static/js/main.830b8bf3.chunk.js:1:30710)
    at https://security.centralpoint.cloud/static/js/main.830b8bf3.chunk.js:1:62218
    at Array.map (<anonymous>)
    at a (https://security.centralpoint.cloud/static/js/main.830b8bf3.chunk.js:1:16089)
    at b (https://security.centralpoint.cloud/static/js/main.830b8bf3.chunk.js:1:62170)
    at u (https://security.centralpoint.cloud/static/js/main.830b8bf3.chunk.js:1:62246)
    at p (https://security.centralpoint.cloud/static/js/main.830b8bf3.chunk.js:1:62641)
```

### Steps to reproduce

Freshly build from GIT:
git clone -b gvm-libs-21.04 https://github.com/greenbone/gvm-libs.git
git clone -b gvmd-21.04 https://github.com/greenbone/gvmd.git
git clone -b gsa-21.04 https://github.com/greenbone/gsa.git
git clone https://github.com/greenbone/openvas-smb.git
git clone -b openvas-21.04 https://github.com/greenbone/openvas.git
git clone -b ospd-21.04 https://github.com/greenbone/ospd.git
git clone -b ospd-openvas-21.04 https://github.com/greenbone/ospd-openvas.git

### GVM versions

Greenbone Security Assistant 21.04.0~git-2cffc292b-HEAD
Greenbone Vulnerability Manager 21.4.0~git-d8d82b405-HEAD
GIT revision d8d82b405-HEAD
Manager DB revision 242
OpenVAS 21.4.0~git-4f11a5e3-HEAD
GIT revision ~git-4f11a5e3-HEAD
gvm-libs 21.4.0~git-4174a549-HEAD

### Environment

**Operating system:**
Linux xxxxxx 5.4.0-74-generic #83-Ubuntu SMP Sat May 8 02:35:39 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=20.04
DISTRIB_CODENAME=focal
DISTRIB_DESCRIPTION="Ubuntu 20.04.2 LTS"

**Comment by bjoernricks** [comment]
Hi thanks for your report. Could you check if this PR is included in your build please https://github.com/greenbone/gsa/pull/2869 ?

**Comment by bjoernricks** [comment]
Also running `gvmd --rebuild` might fix the issue.

**Comment by BasvanH** [comment]
That PR is not in tag release [v21.4.0](https://github.com/greenbone/gsa/releases/tag/v21.4.0), so no it's not in my build.

Have executed `gvmd --rebuild` without succes.

**Comment by y0urself** [comment]
Ok. You wrote, that you checked out 
`git clone -b gvm-libs-21.04 https://github.com/greenbone/gvm-libs.git` not the tag `v21.4.0`
That branch gvm-libs-21.04 should already contain the PR @bjoernricks mentioned.
We will also release 21.4.1 today/tomorrow.

**Comment by BasvanH** [comment]
Oh, sorry, I pasted the wrong git clone command. I did clone it by tags. I will wait for `21.4.1`, thanks.

**Comment by y0urself** [comment]
@BasvanH 21.4.1 is released now.

**Comment by BasvanH** [comment]
Can confirm the 'score' issue has been resolved with [v21.4.1](https://github.com/greenbone/gsa/releases/tag/v21.4.1), thanks!
